packages/ui/lib/components/card-info/CardInfo.tsx

```diff
@@ -0,0 +1,143 @@
+import { Divider, List, ListItem, ListItemText, Stack } from "@mui/material";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import React, { ReactNode } from "react";
+
+export interface CardInfoProps {
+  /** Generic bottom content _(will be displayed after `items`)_ */
+  bottomContent?: ReactNode;
+  /** Main card icon */
+  icon?: ReactNode;
+  /** Structured list of `CardInfoItem` */
+  items?: CardInfoItem[];
+  /** Main card title */
+  title?: string;
+  /** Generic top content _(will be displayed before `items`)_ */
+  topContent?: ReactNode;
+}
+
+export interface CardInfoItemsProps {
+  items: CardInfoItem[];
+}
+
+export interface CardInfoItem {
+  /** footer text _(will be displayed on bottom)_ */
+  footerText?: string;
+  /** Item label */
+  label: string;
+  /** Item value */
+  value: ReactNode | string;
+}
+
+const DATA_TEST_ID_PREFIX = "io-ipatente-card-info";
+
+const CARD_BACKGROUND_COLOR = "#F5F7FB";
+const DIVIDER_BORDER_COLOR = "#CBD5E1";
+const TEXT_PRIMARY_COLOR = "text.primary";
+const TEXT_SECONDARY_COLOR = "#64748B";
+
+/** Basic info card component */
+export const CardInfo = ({
+  bottomContent,
+  icon,
+  items,
+  title,
+  topContent,
+}: CardInfoProps) => (
+  <Card
+    data-testid={DATA_TEST_ID_PREFIX}
+    sx={{ backgroundColor: CARD_BACKGROUND_COLOR, width: "100%" }}
+  >
+    <CardContent sx={{ padding: 2 }}>
+      <Stack
+        alignItems="center"
+        direction="row"
+        fontSize={24} // useful for inherited icon fontSize
+        justifyContent="flex-start"
+        spacing={1}
+      >
+        {icon}
+        <Typography
+          color={TEXT_PRIMARY_COLOR}
+          data-testid={`${DATA_TEST_ID_PREFIX}-title`}
+          fontSize={20}
+          fontWeight={600}
+        >
+          {title}
+        </Typography>
+      </Stack>
+      <Stack
+        alignItems="flex-start"
+        direction="column"
+        justifyContent="flex-start"
+        marginTop={1.5}
+        spacing={1}
+      >
+        {topContent}
+        <CardInfo.Items items={items ?? []} />
+        {bottomContent}
+      </Stack>
+    </CardContent>
+  </Card>
+);
+
+/** Card Info items internal component */
+const Items = ({ items }: CardInfoItemsProps) => (
+  <List
+    data-testid={`${DATA_TEST_ID_PREFIX}-items`}
+    sx={{ padding: 0, width: "100%" }}
+  >
+    {items.map((item, index, items) => (
+      <React.Fragment key={`${DATA_TEST_ID_PREFIX}-item-${index}`}>
```

Comment:
using indexes for keys would [prevent react to optimize rendering](https://react.dev/learn/rendering-lists#why-does-react-need-keys) this list (I don't know if it's the case)

Reply:
Yes right, in our case we should not have any variations during a rendering session.
We expect to have no changes in the number of items, so it might be fine.
@gunzip what do you suggest about this?

Reply:
it's ok as is if you know in advance that all future usage of this component respect this contraints. othwerwise it's better to use another index.